ios/Trifle/Sources/Trifle.swift

```diff
@@ -98,21 +128,37 @@ extension Certificate {
      - returns: true if validated, false otherwise
      */
     public func verify(
-        certificateRequest: MobileCertificateRequest,
-        certificateChain: Array<Certificate>,
-        rootCertificate: Certificate
+        certificateRequest: MobileCertificateRequest?,
```

Comment:
lets add some default values to make the callsite a little cleaner
```
        certificateRequest: MobileCertificateRequest? = nil,
        intermediateChain: Array<Certificate>, // keep this non-nullable
        rootCertificate: Certificate? = nil
```

Making an array nullable creates ambiguity and I think I prefer that an empty list is the default unset value.

Reply:
sounds good.